### PR TITLE
move samples from open_model_zoo

### PR DESCRIPTION
**Pull request by Wovchena**
See https://github.com/opencv/open_model_zoo/pull/118

### CODE REVIEW

**Comment by Wovchena** [comment]
this is supposed to be assigned to @IRDonch

**Comment by IRDonch** [comment]
IMO, it would make sense to rename these from demos to samples. You've already called them samples yourself, and they are in the `sample` folder among other things called samples.

**Comment by Wovchena** [comment]
@IRDonch please, take a look again. I have replaced `demo` with `sample`.

File: inference-engine/ie_bridges/python/sample/dynamic_batch_sample/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Dynamic Batch Python* Sample
+
+This topic demonstrates how to run the 3D Segmentation Demo, which demostrates how to set batch size dynamicly for certain infer request and check inference time difference.
```
[the remark this answers is not in the document]
Comment (Wovchena) [reply inference-engine/ie_bridges/python/sample/dynamic_batch_sample/README.md]: fixed

**Comment by IRDonch** [comment]
Looks good, although I don't know if this is the correct branch to submit this to. You should clarify it with the maintainers of the project.

**Comment by Wovchena** [comment]
@vmatveic is this the correct branch to submit to?

**Comment by vmatveic** [comment]
Hi,
I think yes, but let's ask @asuhov to confirm. Thanks

**Comment by vmatveic** [comment]
@Wovchena could you please clarify what the main goal of this PR? Do I understand correctly that you are removing these demos from the open-model-zoo repo but placing them to this dldt repo as **samples**? If so, then I think you need to implement these changes within our private GitLAB repo first. Because in dldt repo on the 2019 branch we have exact content published from the GitLAB (it's not OMZ repo). Moreover, currently on 2019 branch we can only have 2019R1.1 content officially published but this change is our of this release (most probably for 2019R2). Thanks!

**Comment by Wovchena** [comment]
@vmatveic sorry, for a slow response. You are right, it is a movement from omz to dldt repo. I do not think that this update affects GitLAB repo because it only renames (demo->sample) and moves the existing code, while internal repo has mixed sample/demo layout. Let me know, if I should submit the same renaming in GitLab.

**Comment by vmatveic** [comment]
@Wovchena , yes, if you plan to remove these samples from OMZ then you need to  implement it in our private repo first. Otherwise, these sample will be lost when we will publish R2 content to dldt open source.

**Comment by Wovchena** [comment]
As discussed, this changes will be published later

**Pull request closed without merging**